rclcpp/test/test_timer.cpp

```diff
@@ -0,0 +1,151 @@
+// Copyright 2019 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+
+#include <chrono>
```

Comment:
```
#include <atomic>
```

Currently you're depending indirectly on someone else including that for you which is brittle.

Reply:
BTW you may want to use https://include-what-you-use.org/ to automate that stuff.

`apt-get install iwyu`
`colcon build --cmake-args -DIWYU_LLVM_ROOT_PATH=$(dirname $(which iwyu)) ...`

I can show you tomorrow if you want!

Reply:
Oops, good catch. Will add.